transport-native-epoll/src/main/java/io/netty/channel/epoll/Native.java

```diff
@@ -94,23 +95,29 @@
             // This is ok as strerror returns 'Unknown error i' when the message is not known.
             ERRORS[i] = strError(i);
         }
-        CONNECTION_RESET_EXCEPTION_WRITE = newConnectionResetException("write");
-        CONNECTION_RESET_EXCEPTION_WRITEV = newConnectionResetException("writev");
-        CONNECTION_RESET_EXCEPTION_READ = newConnectionResetException("read");
-        CONNECTION_RESET_EXCEPTION_SENDFILE = newConnectionResetException("sendfile");
-        CONNECTION_RESET_EXCEPTION_SENDTO = newConnectionResetException("sendto");
-        CONNECTION_RESET_EXCEPTION_SENDMSG = newConnectionResetException("sendmsg");
-        CONNECTION_RESET_EXCEPTION_SENDMMSG = newConnectionResetException("sendmmsg");
+        CONNECTION_RESET_EXCEPTION_WRITE = newConnectionResetWriteException("write");
+        CONNECTION_RESET_EXCEPTION_WRITEV = newConnectionResetWriteException("writev");
+        CONNECTION_RESET_EXCEPTION_READ = newConnectionResetReadException("read");
+        CONNECTION_RESET_EXCEPTION_SENDFILE = newConnectionResetWriteException("sendfile");
```

Comment:
I guess it's fine as long as it maps correctly to the corresponding system call names. We might want to add more information to the operation name so that it is clear it's the syscall names. e.g. (`syscall:writev()`)